src/index.html

```diff
@@ -9,10 +9,42 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="./styles/main.scss"
```

Comment:
The HTML file links to a `.scss` file, which browsers cannot interpret. Please compile your SCSS to CSS and update the link to point to the compiled CSS file.